scripts/update_version.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Function to read the version from the VERSION file
+get_version() {
```

Comment:
This function should automatically increase the minor number of the VERSION file.